src/main/java/com/tournament/cup/Start.java

```diff
@@ -0,0 +1,26 @@
+package com.tournament.cup;
+
+import com.tournament.cup.settings.FirstClass;
+import lombok.AllArgsConstructor;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+
+@SpringBootApplication
+@AllArgsConstructor
```

Comment:
If you have `private final` on your bean, it is better to use `lombok.RequiredArgsConstructor`